scripts/validate_releases.sh

```diff
@@ -37,12 +37,11 @@ EXPECTED_XCODEBUILD_RESULTS=( \
     [2.5.3]=${ERROR_CODE} \
     [2.6.0]=${ERROR_CODE} \
     [2.6.1]=${ERROR_CODE} \
+    [2.6.2]=${SUCCESS_CODE} \
     [2.7.0]=${ERROR_CODE} \
     [2.7.1]=${ERROR_CODE} \
     [2.7.2]=${ERROR_CODE} \
-    [2.8.0]=${SUCCESS_CODE} \
-    [2.8.1]=${SUCCESS_CODE} \
-    [2.9.0]=${SUCCESS_CODE}
```

Comment:
It doesn't look like it should be removed.
These are the released versions.

Reply:
PR is not ready for review yet. Accidentally removed draft label.